src/Native/Runtime/unix/PalRedhawkUnix.cpp

```diff
@@ -961,7 +858,32 @@ REDHAWK_PALEXPORT _Ret_maybenull_ _Post_writable_byte_size_(size) void* REDHAWK_
     
     if (allocationType & MEM_RESERVE)
     {
-        void * pRetVal = mmap(pAddress, size, unixProtect, MAP_ANON | MAP_PRIVATE, -1, 0);
+        // For Windows compatibility, let the PalVirtualAlloc reserve memory with 64k alignment.
+        static const size_t Alignment = 64 * 1024;
```

Comment:
Can you file an issue to revisit this with the expected PAL / GC API refactoring?